samza-sql/src/main/java/org/apache/samza/sql/translator/QueryTranslator.java

```diff
@@ -143,10 +182,29 @@ public void init(Config config, TaskContext taskContext) {
 
       @Override
       public void close() {
-
       }
-
     });
+  }
 
+  private void sendToOutputStream(StreamApplicationDescriptor appDesc, TranslatorContext context, RelNode node) {
+    SqlIOConfig sinkConfig = sqlConfig.getOutputSystemStreamConfigsBySource().get(LOG_OUTPUT_STREAM);
+    MessageStreamImpl<SamzaSqlRelMessage> stream = (MessageStreamImpl<SamzaSqlRelMessage>) context.getMessageStream(node.getId());
```

Comment:
nit: we can use MessageStream here instead of MessageStreamImpl, since we don't need to access non-public API functions of stream.

Reply:
done